Tests/Marketplace/configure_and_install_packs.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+from Tests.configure_and_test_integration_instances import Server, set_marketplace_url, MARKET_PLACE_CONFIGURATION,\
+    Build, get_json_file
+from Tests.test_content import ParallelPrintsManager
+from Tests.Marketplace.search_and_install_packs import install_all_content_packs
+from demisto_sdk.commands.common.tools import print_color, LOG_COLORS
+
+
+def options_handler():
+    parser = argparse.ArgumentParser(description='Utility for instantiating and testing integration instances')
+    parser.add_argument('--ami_env', help='The AMI environment for the current run. Options are '
+                                          '"Demisto 6.0", "Demisto Marketplace". The server url is determined by the'
+                                          ' AMI environment.', default="Demisto Marketplace")
+    parser.add_argument('-s', '--secret', help='Path to secret conf file')
+    parser.add_argument('--branch', help='GitHub branch name', required=True)
+    parser.add_argument('--build-number', help='CI job number where the instances were created', required=True)
+
+    options = parser.parse_args()
+
+    return options
+
+
+def main():
+    options = options_handler()
+
+    # Get the host by the ami env
+    hosts, _ = Build.get_servers(ami_env=options.ami_env)
+
+    print_color('Retrieving the credentials for Cortex XSOAR server', LOG_COLORS.NATIVE)
+    secret_conf_file = get_json_file(path=options.secret)
+    username: str = secret_conf_file.get('username')
+    password: str = secret_conf_file.get('userPassword')
+
+    # Configure the Servers
+    for host in hosts:
+        server = Server(host=host, user_name=username, password=password)
+        error_msg: str = 'Failed to set marketplace configuration.'
+        print_color(f'Adding Marketplace configuration to {host}', LOG_COLORS.NATIVE)
+        server.add_server_configuration(config_dict=MARKET_PLACE_CONFIGURATION, error_msg=error_msg)
+        set_marketplace_url(servers=[server], branch_name=options.branch, ci_build_number=options.build_number)
+
+        # Acquire the server's host and install all content packs (one threaded execution)
+        print_color(f'Starting to install all content packs in {host}', LOG_COLORS.NATIVE)
+        server_host: str = server.client.api_client.configuration.host
+        install_all_content_packs(client=server.client, host=server_host, prints_manager=ParallelPrintsManager(1))
```

Comment:
Installation of content packs returns an output of all currecnly installed packs and their versions.

Please make sure you debug the response so that we can know which versions were installed in the build.

Reply:
what info is important for us here?
for each pack we get: ```{'id': 'AMP', 'currentVersion': '1.0.1', 'name': 'Cisco AMP', 'lastInstallDate': '2020-10-25T16:10:02.324339106Z', 'updateAvailable': False}```